views/default.jade

```diff
@@ -0,0 +1,20 @@
+doctype html
```

Comment:
Base template JADE - any other Single Page Application style page will `extend` this - see below.